src/index.ts

```diff
@@ -252,26 +325,58 @@ export class EppoJSClient extends EppoClient {
     subjectKey: string,
     subjectAttributes: Record<string, AttributeType>,
   ): T {
-    EppoJSClient.ensureInitialized();
+    this.ensureInitialized();
     return super.getExperimentContainerEntry(flagExperiment, subjectKey, subjectAttributes);
   }
 
-  private static ensureInitialized() {
-    if (!EppoJSClient.initialized) {
+  private ensureInitialized() {
+    if (!this.initialized) {
+      // TODO: check super.isInitialized?
```

Comment:
`EppoClient.initialized` also exists, but means something different than it does here.

Reply:
what does it mean and how is that different than this? consider renaming either if so